bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Transform.java

```diff
@@ -51,6 +53,10 @@ public class Transform extends Resource {
 	 */
 	public long handle;
```

Comment:
I still see usages of `handle` in the code, can/should they be removed too? If I understand correctly, `handle` should be the handle for the instance in zoom level of the primary monitor so it's "redundant information" that should be obtained via the new method `win32_getHandle(Transform, int)`, right?

If it can't be removed, its visibility could be reduced.
If removing it is possible but it introduces too many changes to the PR, do it in a separate commit so I can review it quickly and you can squash commits at the end.

Let me know which approach you will take.

Reply:
The handle is only used withing the Transform class itself. Its the first handle created wrt to the initialZoomLevel. We keep it for the reference to a particular zoom level. It can also be obtained by calling getHandle(initialZoom), but just keeping the handle field as it is for more readability.
I would reduce the visibility.

Reply:
Removed the handle